tests/test_EEE_module/test_emissions.py

```diff
@@ -0,0 +1,426 @@
+    }
+    observed = em._gather_homegrown_feeds_and_fertilizer_apps()
+
+    assert observed == expected
+    mock_convert_time.assert_called_once_with(2014, 20)
+    mock_filter_variable.assert_called_once_with(time_filter)
+    assert mock_filter_results.call_count == 4
+    mock_filter_results.assert_has_calls(
+        [
+            mocker.call(
+                {
+                    "name": "Homegrown Feeds",
+                    "description": "Collects all crop harvests that occurred in the simulation.",
+                    "filters": ["CropManagement._record_yield.harvest_yield.field='.*'"],
+                    "variables": [".*"],
+                    "date_fields": ("harvest_year", "harvest_day"),
+                },
+                datetime(2014, 1, 20),
+                "harvest_year",
+                "harvest_day",
+            ),
+            mocker.call(
+                {
+                    "name": "Fertilizer Applications",
+                    "description": "Collects all synthetic fertilizer applications that occurred in the simulation.",
+                    "filters": ["Field._record_fertilizer_application\\.fertilizer_application\\.field='.*'"],
+                    "variables": [".*"],
+                    "date_fields": ("year", "day"),
+                },
+                datetime(2014, 1, 20),
+                "year",
+                "day",
+            ),
+            mocker.call(
+                {
+                    "name": "Manure Applications",
+                    "description": "Collects all manure applications that occurred in the simulation.",
+                    "filters": ["Field._record_manure_application\\.manure_application\\.field='.*'"],
+                    "variables": [".*"],
+                    "date_fields": ("year", "day"),
+                },
+                datetime(2014, 1, 20),
+                "year",
+                "day",
+            ),
+            mocker.call(
+                {
+                    "name": "Manure Requests",
+                    "description": "Collects all manure requests that occurred in the simulation.",
+                    "filters": ["Field._record_manure_application\\.manure_request\\.field='.*'"],
+                    "variables": [".*"],
+                    "date_fields": ("year", "day"),
+                },
+                datetime(2014, 1, 20),
+                "year",
+                "day",
+            ),
+        ]
+    )
+
+
+def test_gather_ration_feed_totals(mocker: MockerFixture) -> None:
+    """
+    Test that the totals of feeds from rations given to animals in the last 365 days of the simulation are correct.
+    """
+    em = EmissionsEstimator()
+    mock_filter = mocker.patch.object(em.om, "filter_variables_pool", return_value={"test": {"values": [2, 3, 4]}})
+    expected = {"test": 9.0}
+    observed = em._gather_ration_feed_totals()
+    assert observed == expected
```

Comment:
```suggestion
    expected = {"test": 9.0}
    
    observed = em._gather_ration_feed_totals()
    
    assert observed == expected
```
It helps readability a lot to separate the test into Arrange-Act-Assert sections